app/locales/pt-BR.json

```diff
@@ -0,0 +1,118 @@
+    "notifications_exposure_format_yesterday": "Ontem você cruzou com alguém infectado por {{exposureTime}} minutos. ",
+    "notifications_no_exposure": "Sem informações sobre exposições ao COVID-19 nas últimas duas semanas",
+    "overlap_found_button_label": "Informações públicas carregadas",
+    "overlap_no_results_button_label": "Informações públicas carregadas",
+    "overlap_para_1": "A trilha verde representa sua localização history\n\n O círculo roxo representa o conjunto de dados público.",
+    "overlap_title": "Verificar sobreposição",
+    "push_at_risk_message": "Você cruzou com pacientes com COVID-19",
+    "push_at_risk_title": "Você pode estar em risco",
+    "see_exposure_history": "Ver histórico de exposição",
+    "settings_title": "Painel de controle",
+    "share_location_data": "Compartilhar localização",
+    "show_overlap": "Clique para ver os dados públicos",
+    "team": "Time",
+    "team_para": "Nossa equipe é composta por um consórcio de epidemiologistas, engenheiros, cientistas de dados, evangelistas da privacidade digital, professores e pesquisadores de instituições conceituadas, incluindo: MIT, Harvard, The Mayo Clinic, TripleBlind, EyeNetra, Ernst & Young and Link Ventures.",
+    "terms_of_use": "Termos de uso",
+    "terms_of_use_url": "https://docs.google.com/document/d/1mtdal_pywsKZVMXLHjjj5eKznipPLP8sM1HwFTIhjo0/edit#",
+    "tested_positive_subtitle": "Seus dados privados podem ser transferidos para autoridades de saúde, copiado ou compartilhado.",
+    "tested_positive_title": "Compartilhe seu histórico de localização",
+    "logging_active": "Localização ativada",
+    "logging_inactive": "Localização desativada"
+  },
+  "history": {
+    "no_exposure": "Sem exposição",
+    "possible_exposure_para": "É possível que você tenha entrado em contato ou esteve próximo a alguém que tenha testado positivo para COVID-19",
+    "possible_exposure": "Possível exposição",
+    "timeline": "Linha do tempo",
+    "what_does_this_mean_para": "Baseado no seu histórico de localização, é possível que você tenha estado em contato ou próximo a alguém que foi diagnosticado com COVID-19. Isso não significa que você está infectado, mas você pode estar.\n\nPara mais informações sobre o que você deve fazer, consulte o site da Clínica Mayo.",
+    "what_does_this_mean": "O que isso significa?",
+    "what_if_no_symptoms_para": "Se você não tem sintomas, mas ainda deseja fazer o teste, pode ir ao local de teste mais próximo.\n\nÀs vezes, indivíduos que não apresentam sintomas ainda podem transmitir a infecção e infectar outras pessoas. Manter o distanciamento social e ter cuidado com o contato com grandes grupos ou indivíduos em risco (idosos, aqueles com problemas médicos significativos) é importante para controlar o risco que você e outras pessoas correm.",
+    "what_if_no_symptoms": "E se eu não estiver apresentando sintomas?"
+  },
+
+  "_comment": {
+    "":"Sent by https://www.irrd.org/"
+  }
```

Comment:
This is just a comment but should we have these keys in the file @tstirrat @penrods ?

Reply:
It wont cause any problems, but it might get removed by upcoming translation management systems